src/lib/WireGuard.js

```diff
@@ -54,8 +54,8 @@ module.exports = class WireGuard {
         }
 
         await this.__saveConfig(config);
-        await Util.exec('wg-quick down wg0').catch(() => { });
-        await Util.exec('wg-quick up wg0');
+        await Util.exec(`wg-quick down ${WG_PATH}/wg0.conf`).catch(() => { });
+        await Util.exec(`wg-quick up ${WG_PATH}/wg0.conf`);
```

Comment:
Why is this useful? We're using Docker so who cares about the path :)

Reply:
I am using the container inside homeassistant (stripped down docker). I cannot mount another directory to /etc/wireguard/.